DUI3-DX/Connectors/Autocad/Speckle.Connectors.AutocadShared/HostApp/AutocadLayerManager.cs

```diff
@@ -0,0 +1,113 @@
+using Autodesk.AutoCAD.DatabaseServices;
+using Autodesk.AutoCAD.EditorInput;
+using Autodesk.AutoCAD.LayerManager;
+
+namespace Speckle.Connectors.Autocad.HostApp;
+
+public class AutocadLayerManager
```

Comment:
While this has a specific use the binding together of concrete types creates unnecessarily tight coupling, so I'd like to see an interface here, we can take it with POC: comment to that effect however